pkg/operator/ceph/object/bucket/provisioner.go

```diff
@@ -73,7 +73,15 @@ func (p Provisioner) Provision(options *apibkt.BucketOptions) (*bktv1alpha1.Obje
 		return nil, errors.Wrap(err, "Provision: can't create ceph user")
 	}
 
-	s3svc, err := cephObject.NewS3Agent(p.accessKeyID, p.secretAccessKey, p.getObjectStoreEndpoint(), true, nil)
+	objStore, err := p.getObjectStore()
+	tlsCert := make([]byte, 0)
+	if objStore.Spec.Gateway.SecurePort == p.storePort {
```

Comment:
Can we add this in the Provisioner during the initialization `initializeCreateOrGrant`? So we don't need to call it each time.